compute-coverage.sh

```diff
@@ -14,13 +14,38 @@ set -Eeuxo pipefail
 #    directory containing this script.
 
 # Combine .tix files
+# Avoid tripping on an existing all.tix from a prior run
 SUM_TIX="all.tix"
-hpc sum --output=$SUM_TIX --union --exclude=Main --exclude=GitRev $(find . -name "*.tix")
+hpc sum --output=$SUM_TIX --union --exclude=Main --exclude=GitRev \
+        $(find . ! -path ./all.tix -name "*.tix" -print)
 
-# Generate report
-HPC_ROOT=$(find dist-newstyle -name "hpc")
+# Find the HPC dir, and don't trip on old versions after a version bump.
+# See saw-script #2114.
+#
+# There is no direct way to do this. Instead, fetch the name of the
+# SAW executable. This gives us a path into the build directory for
+# the current version:
+#    dist-newstyle/build/$TARGET/ghc-$GHC/saw-script-$VERSION/build/saw/saw
+#
+# where we don't want to have to try to figure out $TARGET, $GHC, or $VERSION.
+#
+# The hpc dir we want lives under the saw-script-$VERSION dir, but can be in
+# different places with different versions of the tooling. So start there and
+# then use find.
+SAW=$(cabal list-bin -v0 exe:saw)
```

Comment:
It would be worth adding a note referencing https://github.com/GaloisInc/saw-script/issues/2103 in the comment block above to ensure a future developer knows what the `-v0` is for here.

Reply:
Good thought, added in 82362b57bd276660ddd02ceda300aea63e7bbd9b